app/models/npq_registration_api_token.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class NpqRegistrationApiToken < ApiToken
+  attribute :private_api_access, default: true
+
```

Comment:
What does this do?

Reply:
I actual only found this earlier today and mind was blown. its documented here https://api.rubyonrails.org/classes/ActiveRecord/Attributes/ClassMethods.html#method-i-attribute

so it looks like they (rails/activerecord) added a mechanism where it allows for fine grain control between AR and the DB for type casting and general handling of serialisation/unserialisation of types.

in this particular case i'm not using any of that but setting the default value of `private_api_access` to `true` for `NpqRegistrationApiToken` as it defaults to false as defined by the database default value on `ApiToken`

Reply:
So is that only going to impact new npq tokens, or any existing ones too? Mostly a theoretical question, I don't think many of them will survive database wipes.

Reply:
Existing ones will use whatever is persisted in the database. The new ones will take `true`. from the docs:
```
The default value to use when no value is provided. If this option is not passed, the previous default value (if any) will be used. Otherwise, the default will be nil.
```